docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -0,0 +1,305 @@
+---
+---
+
+{% capture overview %}
+
+The goal of this codelab is for you to turn a simple Hello World node.js app 
+into a replicated application running on Kubernetes. We will show you how to 
+take code that you have developed on your machine, turn it into a Docker
+container image, and then run that image on [Minikube](/docs/getting-started-guides/minikube).
+
+Kubernetes is an open source project which can run on many different environments, 
+from laptops to high-availability multi-node clusters, from public clouds to 
+on-premise deployments, from virtual machines to bare metal. Minikube provides
+a simple way of running Kubernetes on your local workstation for free.
+
+{% endcapture %}
+
+{% capture objectives %}
+
+* Run a hello world Node application
+* Deploy the application to Minikube
+* View application logs
+* Update the application image
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* For OS X, you need [Homebrew](https://brew.sh) to install the xhyve
+driver.
+
+* Install Docker. On OS X, we recommend 
+[Docker for Mac](https://docs.docker.com/engine/installation/mac/).
+
+
+{% endcapture %}
+
+{% capture lessoncontent %}
+
+## Create a Minikube cluster
+
+This tutorial uses [Minikube](https://github.com/kubernetes/minikube) to 
+create a local cluster. This tutorial also assumes you are using [Docker for Mac](https://docs.docker.com/engine/installation/mac/)
+on OS X. If you are on a different platform like Linux, or using VirtualBox
+instead of Docker for Mac, the instructions to install Minikube may be
+slightly different. For general Minikube installation instructions, see
+the [Minikube installation guide](docs/getting-started-guides/minikube/).
+
+### Download and install the Minikube release
+
+Use `curl` to download and install the latest Minikube release:
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+```
+
+Use Homebrew to install the xhyve driver and set its permissions.
+
+```shell
+brew install docker-machine-driver-xhyve
+sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+```
+
+Download the latest version of the `kubectl` command-line tool, which you can
+use to interact with Kubernetes clusters.
+
+```shell
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
+Start the Minikube cluster:
+
```

Comment:
minikube start --vm-driver=xhyve
I get Santa, This application has been blocked from executing because its trustworthiness cannot be determined. Filename: minikube, Path: /usr/local/bin/minikube, Publisher: Not code-signed, etc.